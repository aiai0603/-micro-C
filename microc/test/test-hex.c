Void main(Int n) {
  
    Int a;
    a = createI("F122",16);

    print(16,a);

    Int b;
    b = createI("1010",2);

    print(2,b);

}

