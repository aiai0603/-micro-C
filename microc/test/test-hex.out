24 19 1 5 25 15 1 13 0 1 1 0 61730 12 15 -1 13 0 1 1 11 22 13 0 2 1 0 10 12 15 -1 13 0 2 1 11 22 21 3